config/routes.rb

```diff
@@ -1,11 +1,11 @@
 Rails.application.routes.draw do
-  resources :comments
-  resources :likes
-  resources :posts
-  resources :users
-  
-  root 'users#index'
+    root 'users#index'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :new, :show] do
+      resources :comments, only: [:new, :create] 
+      resources :likes, only: [:create] 
+    end
   end
+  # post '/posts', to: 'posts#create'
```

Comment:
- [ ] Great job working on this milestone. In to maintain our repo in a professional way, we should always deleted all commented lines of code. We can always re-visit old codes with git history